src/components/dashboard/Usage.vue

```diff
@@ -51,70 +51,89 @@ const allLimits = computed(() => {
 })
 
 async function getAppStats() {
-  const date_id = new Date().toISOString().slice(0, 7)
   if (!main.user)
     return { data: [], error: 'missing user' }
 
   if (props.appId) {
-    // console.log('appID', props.appId)
     return supabase
-      .from('app_stats')
+      .from('app_usage')
       .select()
-      .eq('user_id', main.user?.id)
       .eq('app_id', props.appId)
-      .like('date_id', `${date_id}%`)
   }
   else {
     return supabase
-      .from('app_stats')
+      .from('app_usage')
       .select()
-      .eq('user_id', main.user?.id)
-      .like('date_id', `${date_id}%`)
   }
 }
 
 async function getAllStats() {
-  // get aapp_stats
   if (!main.user?.id)
     return
+
   const date_id = new Date().toISOString().slice(0, 7)
   stats.value = await getTotalStats(main.user?.id, date_id)
 }
+
 async function getUsages() {
+  const currentStorage = 0
   const { data, error } = await getAppStats()
   if (data && !error) {
     datas.value.mau = Array.from({ length: getDaysInCurrentMonth() }).fill(undefined) as number[]
```

Comment:
getDaysInCurrentMonth should be modified to get the number of day between the start and end of cycle.
Chart should now display day of cycle not of the month

Reply:
resolved